atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/TransactionConfig.java

```diff
@@ -51,6 +52,15 @@ public boolean lockImmutableTsOnReadOnlyTransactions() {
         return false;
     }
 
+    /**
+     * Indicates how long user transactions are allowed to take to commit, in terms of how long we'll refresh the
+     * commit locks for.
+     */
+    @Value.Default
+    public HumanReadableDuration commitLockTenure() {
+        return HumanReadableDuration.minutes(5);
+    }
```

Comment:
The server can still hold the lock for 10 more minutes (maxAllowedLockTimeout) , right? Is so, maybe worth documenting that as well.
Actually I think releasing the locks sooner is also not a bad idea.

Reply:
The 10 minutes is for the v1 lock service. These commit locks should be v2 ones, which I think by default expire in 20 seconds (LockLeaseContract). It's fair to note that more time may be needed - I've added a line to the comment suggesting that.